scripts/auto-rebase/rebase.sh

```diff
@@ -884,7 +910,7 @@ update_changelog() {
          test -n "$(git status -s scripts/auto-rebase/changelog.txt scripts/auto-rebase/commits.txt)" && \
          title "## Committing changes to changelog" && \
          git add scripts/auto-rebase/commits.txt scripts/auto-rebase/changelog.txt && \
-         git commit -m "update changelog" || true)
+         git commit -m "update changelog") || true
```

Comment:
This was error https://www.shellcheck.net/wiki/SC2015

Was the intention here `(this AND that) OR true`?